### PR TITLE
IPRO-17:add_signup_page

### PR DESCRIPTION
**Pull request by MbiydzenyuyStephen**

### CODE REVIEW

File: src/components/signup.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+
+class Signup extends Component {
```
Comment (Lambertyubin) [inline src/components/signup.jsx:4]: Please use functional components instead of class components

File: src/index.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import Signup from './components/signup';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
```
Comment (Lambertyubin) [inline src/index.js:5]: You need to remove this

File: src/index.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import Signup from './components/signup';
 import './index.css';
```
Comment (Lambertyubin) [inline src/index.js:4]: Are you using this?

**Review by Lambertyubin — CHANGES_REQUESTED** [review]

**Review by Lambertyubin — APPROVED** [review]

File: src/components/signup.jsx (hunk 1 above)
Comment (MbiydzenyuyStephen) [reply src/components/signup.jsx]: Changes made

File: src/index.js (hunk 3 above)
Comment (MbiydzenyuyStephen) [reply src/index.js]: Only line 4 wasnt used

**Pull request merged**